distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -93,14 +93,88 @@ export project.System.Platform
 export project.System.Process
 export project.System.Process.Exit_Code.Exit_Code
 export project.Warning.Warning
-from project.Data.Boolean export Boolean, False, True
-from project.Data.Json.Extensions export all
-from project.Data.Numbers export Float, Integer, Number
-from project.Data.Range.Extensions export all
-from project.Data.Statistics.Extensions export all
-from project.Data.Text.Extensions export all
-from project.Data.Text.Regex export regex
-from project.Function export all
-from project.Meta.Enso_Project export enso_project
-from project.Network.Extensions export all
-from project.System.File_Format export Auto_Detect, Bytes, File_Format, Infer, JSON_Format, Plain_Text_Format
+export project.Data.Boolean.Boolean
```

Comment:
Is `from ... export` deprecated even when it's not using `all`?

Reply:
They are not deprecated, they are just now suffering from https://github.com/enso-org/enso/issues/10399. Until that issue is fixed, rule of a thumb is to just use plain `export ...`.